src/MOI/MOI_wrapper.jl

```diff
@@ -2515,6 +2551,9 @@ end
 
 function MOI.get(model::Optimizer, attr::MOI.RawStatusString)
     _throw_if_optimize_in_progress(model, attr)
+    if haskey(_ERROR_TO_STATUS, model.ret_optimize)
+        return _get_error_string(model.env, model.ret_optimize)
```

Comment:
Have you tested this works when called at anytime? I.e., does it depend on some internal state of the `env`?

Reply:
```Julia
julia> env = CPLEX.Env()
CPLEX.Env(Ptr{Nothing} @0x00007f9c942a1010, false, 0)

julia> CPLEX._check_ret(env, Cint(CPLEX.CPXERR_NO_MEMORY))
ERROR: CPLEX Error  1001: Out of memory.

Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] _check_ret(::CPLEX.Env, ::Int32) at /Users/oscar/.julia/dev/CPLEX/src/MOI/MOI_wrapper.jl:126
 [3] top-level scope at REPL[8]:1
```

What's weird is
```Julia
julia> CPLEX._check_ret(C_NULL, Cint(CPLEX.CPXERR_NO_MEMORY))
ERROR: CPLEX Error  1001: Out of memory.

Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] _check_ret(::Ptr{Nothing}, ::Int32) at /Users/oscar/.julia/dev/CPLEX/src/MOI/MOI_wrapper.jl:126
 [3] top-level scope at REPL[9]:1
```
Why even have the pointer as the argument?

Reply:
Well, it is developed by IBM. I would guess that not taking `env` would be considered as exposing the internals, because they would be exposing `env` is not needed. This way if they somehow start needing some info on `env` the call already takes the object.